tests/test_cwe.py

```diff
@@ -51,6 +51,42 @@ def test_event_rule_tags(self):
                         'Tags')}
         self.assertEqual(tags, {'App': 'Custodian'})
 
+    def test_event_rule_enable(self):
+        factory = self.replay_flight_data('test_cwe_enable_rule')
+        policy = self.load_policy(
+            {
+                'name': 'cwe-enable-rule',
+                'resource': 'aws.event-rule',
+                'actions': [
+                    {
+                        'type': 'set-rule-state',
+                        'enabled': True
+                    }
+                ]
+            },
+            session_factory=factory,
+        )
+        resources = policy.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
you're missing assertions in these tests to check that the rules have actually been enabled or disabled

Reply:
Updated as suggested